response_operations_ui/views/messages.py

```diff
@@ -188,6 +188,17 @@ def view_conversation(thread_id):
                 thread_data=thread_conversation,
             )
 
+    close_conversation_url = url_for(
```

Comment:
I don't like this function, however also hate it on the frontend, the whole function definitely needs a refactor as it's difficult to follow